Sources/MeiliSearch/QueryParameters/DocumentsQuery.swift

```diff
@@ -0,0 +1,21 @@
+import Foundation
+
+public class DocumentsQuery: Queryable {
+  private var limit: Int?
+  private var offset: Int?
+  private var fields: [String]
+
+  init(limit: Int? = nil, offset: Int? = nil, fields: [String]? = nil) {
+    self.offset = offset
+    self.limit = limit
+    self.fields = fields ?? []
+  }
+
+  internal func buildQuery() -> [String: Codable?] {
+    [
+      "limit": limit,
+      "offset": offset,
+      "fields": fields.isEmpty ? nil : fields.joined(separator: ",")
```

Comment:
I answered here: https://github.com/meilisearch/meilisearch-swift/pull/312#discussion_r936606588

Reply:
but indeed, there is an error in this logic, because the fields can be empty! It should be `"fields": fields != nil ? fields?.joined(separator: ",") : nil`